modules/sso/install

```diff
@@ -10,7 +10,16 @@ pushd $JBOSS_HOME/bin
 ./jboss-cli.sh --file=keycloak-install.cli
 popd
 
+# Clean up the left-over content of the history directory if it exists
+XML_HISTORY_CURRENT_DIR="$JBOSS_HOME/standalone/configuration/standalone_xml_history/current"
+if [ -d "$XML_HISTORY_CURRENT_DIR" ]; then
```

Comment:
Generally speaking you don't need to check if the dircetory exists if you use `rm -rf`. I suggest to remove it and just create a one-liner :)